### ISSUE TITLE
clarifying the relationship between rule application and accessibility conformance

### ISSUE THREAD

**Issue by jasonjgw** [w3c/wcag-act#227]
I agree with the note in the draft which suggests that using the terms "pass" and "fail" with respect both to individual conformance testing rules and to accessibility conformance itself is confusing. In particular, "passing" an accessibility conformance testing rule may result in "failing" to conform to an accessibility requirement asserted in WCAG or elsewhere. "Failing" an accessibility conformance testing rule may contribute to "passing" an accessibility conformance requirement.

I suggest avoiding the terms "pass" and "fail" in connection with ACT rules by using terms such as "satisfying" and "not satisfying" a rule. A rule is "satisfied" or "not satisfied" (alternatively, if you prefer, "unsatisfied") - never "passed" or "failed". Then we can say, without contradiction, that "satisfying" an ACT rule may result in "failing" to meet an accessibility conformance requirement (e.g., a success criterion) - and so forth. Such an explanation in these terms could usefully be included in the text.

**Comment by moekraft** [comment]
From 8/2 Meeting Minutes:
wilco: pass and fail directly inherit from EARL

shadi: so technically WCAG does not define pass and fail, it talks about satisfying the requirements, or meeting the requirements
... I did put some editorial suggestions that I think will help this. We often use pass and fail. I think pass fail for rules, and satisfy/not satify for requirements

wilco: How would we then better address this confusion?

shadi: Instead of using satisfy and not satisfy for rules. In this way you could pass a rule, but not satisfy a requirement

wilco: when it comes to success criteria, most professionals do pass/fail/inapplicable, so your proposal is not the common language

kathy: So my understanding is that with the atomic rules you can pass or fail, and the composed rules and determine if it meets the requirement

wilco: technically speaking you can never pass a success criteria, you can just not satisfy. Its more towards making a best effort to show you do not have the listed problems

alistair: there are only a few instances where we have said a success criteria can be passed or met. why don't we just specify our language somewhere that a success criteria has been met/unmet
... i think just changing the language a bit will fix

shadi: Agreed, i think we just need to spell things out and make sure we are clear.

wilco: Assigning shadi to this issue

**Comment by WilcoFiers** [comment]
Related to #219
